semantic-conventions/src/opentelemetry/semconv/model/semantic_attribute.py

```diff
@@ -358,6 +378,22 @@ def is_simple_type(attr_type: str):
             "boolean[]",
         )
 
+    @staticmethod
+    def is_template_type(attr_type: str):
+        if not isinstance(attr_type, str):
+            return False
+        matchObj = TEMPLATE_TYPE_RE.fullmatch(attr_type)
+        if matchObj is not None:
+            return AttributeType.is_simple_type(matchObj.group(1))
+        return False
+
+    @staticmethod
+    def get_core_template_type(attr_type: str):
+        matchObj = TEMPLATE_TYPE_RE.fullmatch(attr_type)
+        if matchObj is not None:
+            return matchObj.group(1)
+        raise ValidationError(0, 0, f"Cannot retrieve core type from type: {attr_type}")
```

Comment:
I don't remember anymore: I the 0, 0 caught and reraised with the proper line number later? Or should we rather pass on the location to fill it out here?

Reply:
Actually, we don't need it here, because it will be checked during parsing anyways.